src/components/AdminPage/TopMenu.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { Menu } from 'semantic-ui-react';
+import maiLogo from '../Logo';
```

Comment:
The logo should be stored in the `assets` folder.

Reply:
I created a separate file called Logo.js which creates the Logo because i wasn't able to call the Logo using '../Assets/maiLogo.js' it kept giving me errors saying file not found

Reply:
I misunderstood. Assets should only contain things like images, not js files. You're good 👌🏼